common/common-rest/src/main/java/org/apache/servicecomb/common/rest/filter/inner/ServerRestArgsFilter.java

```diff
@@ -50,7 +54,7 @@ public Response afterReceiveRequest(Invocation invocation, HttpServletRequestEx
   }
 
   @Override
-  public void beforeSendResponse(Invocation invocation, HttpServletResponseEx responseEx) {
+  public CompletableFuture<Void> asyncBeforeSendResponse(Invocation invocation, HttpServletResponseEx responseEx) {
```

Comment:
I think  beforeSendResponseAsync is a better name

Reply:
done.